xpost_comments.php

```diff
@@ -19,9 +19,11 @@
 along with this program; if not, write to the Free Software
 Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 */
+include 'wordpress/wp-load.php';
 
 require_once( 'xpost_config.php' );
 require_once( ABSPATH . WPINC . '/class-IXR.php' );
```

Comment:
Are the lines with `class-IXR.php` still required given that `IXR_Client` has been replaced with `WP_HTTP_IXR_CLIENT`?

Reply:
I've based the modifications on this comment from Stackoverflow https://stackoverflow.com/questions/29290852/wordpress-xmlprc-not-working-in-https
Anyway, when I remove the class-IXR line, all hell breaks lose and wp-load.php does not work anymore. 

However, I am no PHP expert (last time I developed something with PHP was before git ;-)

Reply:
I see.